pkg/asset/installconfig/powervs/dns.go

```diff
@@ -27,21 +28,20 @@ func GetDNSZone() (*Zone, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
 	defer cancel()
 
-	publicZones, err := client.GetDNSZones(ctx)
+	// Default to offering only external DNS entries, as IBM Cloud does in their provider
+	// TODO(mjturek): Offer private zones (IBM DNS) when deploying a private cluster
```

Comment:
it might be good to make this a task for a future release vs a todo

Reply:
Agreed. Should it be in the 4.13 epic?

Reply:
yah. not sure if we have one yet though